1-3.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "[DRAFT] Unit 3: Business Case and Benefits"
+permalink: /curricula/1-3/
+ref: /curricula/1-3/
+lang: en
+github:
+  repository: w3c/wai-curricula
+  path: 1-3.md
+footer: >
+  <p>Work in Progress.</p>
+  <p><strong>Date:</strong> $date </p>
+  <p><strong>Editors:</strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed as part of the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
+navigation:
+  previous: /curricula/1-2/
+  next: /curricula/1-4/
+---
+
+## Introduction
+{:.no-display}
+ 
+Courses based on this unit are expected to:
+* examine the rationale for organizations to address accessibility
+* present different testimonials from organizations that have addressed accessibility
+
+Note: Introduce different case studies for accessibility in organizations. Select them as a starting point for students to build their own business case arguments. Use the resource [The Business Case for Digital Accessibility](https://www.w3.org/wai/business-case/).
```

Comment:
This statement seems very instructive without any indication of how to do that. Maybe add something "Research relevant case studies on accessibility in organizations that are relevant to your audience"? Seems like a tall order in any case (where to research and how?)